CS/Services/AIAssistantManager.cs

```
@@ -40,9 +46,28 @@ public AIAssistantCreator(OpenAIClient client, string deployment) {
             
 
             ClientResult<Assistant> assistantResponse = await assistantClient.CreateAssistantAsync(deployment, assistantCreationOptions, ct);
+            var assistant = assistantResponse.Value;
             ClientResult<AssistantThread> threadResponse = await assistantClient.CreateThreadAsync(cancellationToken: ct);
+            var thread = threadResponse.Value;
+
+            return new(assistant.Id, thread.Id, file.Id);
+        }
+        
+        public async Task CleanUpAssistantAsync(AIAssistantData assistantData) {
+            try{
+                if(!string.IsNullOrEmpty(assistantData.AssistantId)){
+                    await assistantClient.DeleteAssistantAsync(assistantData.AssistantId);
+                }
+
+                if(!string.IsNullOrEmpty(assistantData.ThreadId)){
+                    await assistantClient.DeleteThreadAsync(assistantData.ThreadId);
+                }
 
-            return (assistantResponse.Value.Id, threadResponse.Value.Id);
+                if(!string.IsNullOrEmpty(assistantData.FileId)){
+                    await fileClient.DeleteFileAsync(assistantData.FileId);
+                }
+            }
+            catch{}
```

I'd rather log the error. Just swallow an exception is a bad practice. I know that we have the same code in our codebase.